integration_tests/packages.yml

```diff
@@ -1,11 +1,80 @@
 packages:
-  - package: fivetran/twitter_ads
-    version: [">=0.6.0", "<1.0.0"]
-  - package: fivetran/zendesk
-    version: [">=0.10.0", "<1.0.0"]
+  - package: fivetran/ad_reporting
+    version: [">=1.0.0", "<2.0.0"]
+
+  - package: fivetran/shopify_holistic_reporting
+    version: [">=0.1.0", "<1.0.0"]
+
+  ## To be switched after the social media integration tests is updated
+  # - package: fivetran/social_media_reporting
+  #   version: [">=0.1.0", "<1.0.0"]
+  - git: https://github.com/fivetran/dbt_social_media_reporting.git
+    revision: MagicBot/integation-test-webhooks-14
+    warn-unpinned: false 
+
+  - package: fivetran/youtube_analytics
+    version: [">=0.1.0", "<1.0.0"]
+
+  - package: fivetran/app_reporting
+    version: [">=0.1.0", "<1.0.0"]
+
+  - package: fivetran/salesforce
+    version: [">=0.1.0", "<1.0.0"]
   - package: fivetran/hubspot
-    version: [">=0.8.0", "<1.0.0"]
-  - package: fivetran/jira
-    version: [">=0.11.0", "<1.0.0"]
+    version: [">=0.1.0", "<1.0.0"]
+
+  - package: fivetran/shopify
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/stripe
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/recurly
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/zuora
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/recharge
+    version: [">=0.1.0", "<1.0.0"]
+
+  - package: fivetran/marketo
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/iterable
+    version: [">=0.1.0", "<1.0.0"]
   - package: fivetran/klaviyo
-    version: [">=0.5.0", "<1.0.0"]
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/mailchimp
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/pardot
+    version: [">=0.1.0", "<1.0.0"]
+
+  - package: fivetran/quickbooks
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/netsuite
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/sage_intacct
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/xero
+    version: [">=0.1.0", "<1.0.0"]
+
+  - package: fivetran/fivetran_log
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/lever
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/greenhouse
+    version: [">=0.1.0", "<1.0.0"]
+
+  - package: fivetran/amplitude
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/pendo
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/mixpanel
+    version: [">=0.1.0", "<1.0.0"]
+
+  - package: fivetran/jira
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/github
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/asana
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/zendesk
+    version: [">=0.1.0", "<1.0.0"]
+  - package: fivetran/intercom
+    version: [">=0.1.0", "<1.0.0"]
```

Comment:
I actually did a quick assessment of that package for this coverage. It turns out the [only macro we use from fivetran_utils ](https://github.com/search?q=repo%3Afivetran%2Fdbt_salesforce_formula_utils%20fivetran_utils&type=code)in that package is the `fivetran_utils.drop_schemas_automation` macro and it is only used in our integration tests pipeline. Further, the package is structured quite differently from all the others that it would require a pretty customer testing pipeline. 

After doing the assessment I felt it was more work to get that package included than the value it would provide. I do think we should consider testing this further in the future, but felt it wasn't a necessary component for this PR.